.gitignore

```diff
@@ -23,3 +23,5 @@ _yardoc
 doc/
 .idea/
 vendor
+workspace.code-workspace
+.markdownlint.json
```

Comment:
These are configuration settings for a common markdown linter. I will write up a proposal and share it will the team. I am ignoring them for now.